packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.tsx

```diff
@@ -10,13 +10,14 @@ const WalletsListingRoute: React.FC = () => {
     const { switchAccount } = useAuthorize();
     const { data: activeWallet } = useActiveWalletAccount();
 
-    const firstLoginid = walletAccounts?.[0]?.loginid;
-
     useEffect(() => {
-        if (!activeWallet && firstLoginid) {
-            switchAccount(firstLoginid);
+        if (!activeWallet) {
```

Comment:
I think we still need the first login id, because when we first time login, we go straight to wallets page

Reply:
There would still be an active linked account right ?
We are switching to that wallet.

Reply:
I think we should check for the trading account in localstorage and find the linked wallet and switch it here as a fallback. and also have the first wallet as the last fallback if any of those fails. Because in QA box first time login we had this issue

Reply:
@sergei-deriv @thisyahlen-deriv Actually activeLinkedAccount is the active account from local storage, since we are using the local storage values in useAuthorize and login id from useAuthorize is used to select the activeLinkedAccount.
I have also added the logic to switch to first wallet as the fallback.